gcloud/resource_manager/project.py

```diff
@@ -0,0 +1,323 @@
+        and there is no client set as an instance variable, we'll raise a
+        :class:`gcloud.exceptions.MissingClientError`.
+
+        :type client: :class:`gcloud.resource_manager.client.Client`
+        :param client: An optional client to test for existence.
+        """
+        client = client or self.client
+        if not client:
+            raise MissingClientError()
+        return client
+
+    def create(self, client=None):
+        """API call:  create the project via a ``POST`` request.
+
+        Example::
+
+            >>> from gcloud import resource_manager
+            >>> client = resource_manager.Client()
+            >>> project = client.project('new-spaceship-123')
+            >>> project.name = 'New Spaceship Project!'
+            >>> project.create()
+
+        See
+        https://cloud.google.com/resource-manager/reference/rest/v1beta1/projects/create
+
+        :type client: :class:`gcloud.resource_manager.client.Client` or None
+        :param client: the client to use.  If not passed, falls back to
+                       the ``client`` attribute.
+        """
+        client = client or self.client
+        data = {'projectId': self.project_id, 'name': self.name,
+                'labels': self.labels}
+        resp = client.connection.api_request(method='POST', path='/projects',
+                                             data=data)
+        self.set_properties_from_api_repr(resource=resp)
+
+    def reload(self, client=None):
+        """API call:  reload the project via a ``GET`` request.
+
+        This method will reload the newest metadata for the project.
+
+        .. warning::
+
+            This will overwrite any local changes you've made and not saved!
+
+        Example::
+
+            >>> from gcloud import resource_manager
+            >>> client = resource_manager.Client()
+            >>> project = client.get_project('purple-spaceship-123')
+            >>> project.name = 'Locally changed name'
+            >>> print project
+            <Project: Locally changed name (purple-spaceship-123)>
+            >>> project.reload()
+            >>> print project
+            <project: Purple Spaceship (purple-spaceship-123)>
+
+        See
+        https://cloud.google.com/resource-manager/reference/rest/v1beta1/projects/get
+
+        :type client: :class:`gcloud.resource_manager.client.Client` or None
+        :param client: the client to use.  If not passed, falls back to
+                       the ``client`` attribute.
+        """
+        client = self._require_client(client=client)
+
+        # We assume the project exists. If it doesn't it will raise a NotFound
+        # exception.
+        resp = client.connection.api_request(method='GET', path=self.path)
+        self.set_properties_from_api_repr(resource=resp)
+
+    def update(self, client=None):
+        """API call:  update the project via a ``PUT`` request.
```

Comment:
Yep, there is no patch method on this API (https://cloud.google.com/resource-manager/reference/rest/v1beta1/projects/update) :-/